platform/src/test/java/org/exoplatform/platform/qa/ui/platform/plf/PlfBrandingTestIT.java

```diff
@@ -0,0 +1,157 @@
+      manageLogInOut.signOut();
+    }
+    manageLogInOut.signInCas(username, password);
+  }
+
+  @AfterEach
+  public void signout() {
+    manageLogInOut.signOut();
+  }
+
+  /**
+   * <li>Case ID:120887.</li>
+   * <li>Test Case Name: Select navigation bar style.</li>
+   * <li>Pre-Condition:</li>
+   * <li>Post-Condition:</li> Step Number: 1 Step Name: Open branding page Step
+   * Description: - Login as admin - Connect to intranet home page - Go to
+   * Administration button -> Portal -> Branding Input Data: Expected Outcome: -
+   * Branding portlet is displayed Step number: 2 Step Name: Show a theme select
+   * box Step Description: - Go to the part to select the theme - Click on the
+   * select box Input Data: Expected Outcome: - The select box is opened with two
+   * choices:* Dark* Light Step number: 3 Step Name: Select one theme Step
+   * Description: - Select one from the list Input Data: Expected Outcome: - The
+   * full navigation bar preview is displaying the selected style navigation
+   * barover the navigation bar Step number: 4 Step Name: Save Step Description: -
+   * Click on save button Input Data: Expected Outcome: - The navigation bar of
+   * the intranet is displayed as preview
+   */
+
+  @Test
+  public void test01_SelectNavigationBarStyle() {
+    info("Test 1: Select navigation bar style");
+
+    navigationToolbar.goToBanding();
+    waitForAndGetElement(ELEMENT_PLF_BRANDINGPAGE);
+
+    click(ELEMENT_PLF_BRANDING_SELECTTHEME);
+    waitForAndGetElement(ELEMENT_PLF_BRANDING_THEMELIGHT, 2000, 0);
+    waitForAndGetElement(ELEMENT_PLF_BRANDING_THEMEDARK, 2000, 0);
+
+    click(ELEMENT_PLF_BRANDING_THEMELIGHT);
+    waitForAndGetElement(ELEMENT_PLF_BRANDING_TOPBAR_THEMELIGHT, 2000, 0);
+
+    ELEMENT_BUTTON_APPLY.click();
+    waitForAndGetElement(ELEMENT_TOOLBAR_THEMELIGHT, 2000, 0);
+    $(ELEMENT_PLF_BRANDING_SELECTTHEME).click();
+    $(ELEMENT_PLF_BRANDING_THEMEDARK).click();
+    ELEMENT_BUTTON_APPLY.click();
+  }
+
+  /**
+   * <li>Case ID:120888.</li>
+   * <li>Test Case Name: Select logo.</li>
+   * <li>Pre-Condition:</li>
+   * <li>Post-Condition:</li> Step Number: 1 Step Name: Open branding page Step
+   * Description: - Login as admin - Connect to intranet home page - Go to
+   * Administration button -> Portal -> Branding Input Data: Expected Outcome: -
+   * Branding portlet is displayed Step number: 2 Step Name: Upload a logo Step
+   * Description: - Upload a new logo image Input Data: Expected Outcome: - The
+   * new logo image is uploaded Step number: 3 Step Name: Preview Step
+   * Description: - Check preview boxes Input Data: Expected Outcome: - The logo
+   * must be displayed in :* preview boxe for the logo* preview of the full
+   * navigation bar
+   */
+
+  // done in the upload function
+
+  /*
+   * Step number: 4 Step Name: Save Step Description: -Click on Save Input Data:
+   * Expected Outcome: New Logo is shown as preview
+   */
```

Comment:
Several blocks of comments for the same method.

Reply:
changes done